src/main/java/com/matag/game/cardinstance/CardInstance.java

```diff
@@ -195,6 +195,10 @@ public void checkIfCanBlock(CardInstance blockedCreature) {
         throw new MessageException(getIdAndName() + " cannot block " + blockedCreature.getIdAndName() + " as it has flying.");
       }
     }
+
+    if (blockedCreature.hasAbilityType(AbilityType.MENACE) && blockers.size() < 1) {
```

Comment:
This seems fishy... isn't it < 2?

Reply:
interesting... seems like this is excluding the main blocker.

Not sure I understand it yet.